src/commonMain/kotlin/org/intellij/markdown/flavours/gfm/lexer/_GFMLexer.kt

```diff
@@ -1,14 +1,13 @@
 /* The following code was generated by JFlex 1.7.0 tweaked for IntelliJ platform */
 package org.intellij.markdown.flavours.gfm.lexer
 
-import org.intellij.markdown.lexer.GeneratedLexer
 import org.intellij.markdown.MarkdownTokenTypes
 import org.intellij.markdown.IElementType
 import kotlin.Throws
 import org.intellij.markdown.flavours.gfm.GFMTokenTypes
-import org.intellij.markdown.lexer.Compat
-import org.intellij.markdown.lexer.pop
-import org.intellij.markdown.lexer.push
+import org.intellij.markdown.html.BitSet
+import org.intellij.markdown.html.IntStack
+import org.intellij.markdown.lexer.*
```

Comment:
Yes, this is pure manual update of lexer generated files. I was not sure what is the process of generating them and I'm not sure it will be possible to keep all things autometed.